src/common/m_precision_select.f90

```diff
@@ -0,0 +1,17 @@
+!>
+!! @file m_precision_select.f90
+!! @brief Contains module m_precision_select
+
+!> @brief This file contains the definition of floating point used in MFC
+module m_precision_select
+    use mpi
```

Comment:
@aricer123 if you wrap the `use mpi` like this in an `ifdef`,
```fortran
#ifdef MFC_MPI
    use mpi                    !< Message passing interface (MPI) module
#endif
```
it should pass the `--no-mpi` workflow. MFC supports being built with and without MPI, the library used to scale to multiple cores/nodes.

Reply:
yea i figured because it failed on the test with no mpi

Reply:
there was another issue with syscheck module i fixed too will commit another draft soon

Reply:
Great!